### ISSUE TITLE
[BottomAppBar] VoiceOver failure

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#3165]
b/74076822

The BottomAppBar example is reading out image names (e.g., "ic_search" and "ic_menu") instead of an accessibilityLabel. The example should set explicit accessibilityLabel values.

Hamburger menu: "Close"
Search: "Sample Button" (?)

**Comment by romoore** [comment]
The internal issue is for a specific client app and the fixes should be applied there, instead. The Bottom App Bar does not have default actions and so cannot provide accessibility values or labels.

**Comment by romoore** [comment]
Internal issue was resolved.
